### ISSUE TITLE
Accessibility / semantic HTML selectors missing 

### ISSUE THREAD

**Issue by georgegrigorita** [Trevald/WhatTheTag.com#35]
I found this app via an old article in Smashing Magazine and I thought, here's a good idea for when we don't know if we should use <section> or <article>. And instead of reading a [36min article](https://www.smashingmagazine.com/2022/07/article-section-elements-accessibility/) about this on Smashing Magazine I can just open this website and make a decision based on what I need. But then I found that the semantic HTML selectors are not included. It would make a nice addition to it, I think.

**Comment by Trevald** [comment]
Hey, sry for the late answer. Had to refactor a lot to make this app work again. Great input! Semantic selectors is something I’ve been struggling myself with understanding when to use (or rather when not to use).

The app follows a basic tree structure to end up at a recommendation. If you have any ideas for simple and short questions that could work; please feel free to share them here. Either just here in the thread or with a PR.

In the meantime I’ll read up on semantic selectors and see if I can come up with some simple way to get them in.

Cheers!
